sandbox-templates/mdx-evaluate-create-react-app/src/App.js

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import * as runtime from "react/jsx-runtime";
```

Comment:
If we’re going to keep versions locked, and given that we typically don’t update `.github` repos often, I can already see folks trying to reproduce things on old versions.

Maybe, at the start of these JS files, add a big note:

```
// ⚠️ Important! Please make sure the dependencies are up to date.
// You can refresh them in the Dependencies section (left-bottom on CodeSandbox)
```

Reply:
noted here and in next.config.js

Reply:
plus renovate bot could also help if wanted